### PR TITLE
tests: fix cases

### PR DESCRIPTION
**Pull request by nickkbell**

### CODE REVIEW

**Review by DarkPark — APPROVED** [review]

**Review by aleks-and — DISMISSED** [review]

File: tests/objects/MainPage.js [hunk 1]
```diff
@@ -1,11 +1,51 @@
...
+    formatDump ( dump ) {
+        return dump.replace(/(.{2})/g, '$1 ').trim();
+    }
+
+    getDumpInLogs ( dump ) {
+        return this.page.locator('[data-test="hex"]', {hasText: dump}).first();
+    }
+
+    redirectToGithub () {
+        return this.page.getByRole('link', {name: 'GitHub'})
```
Comment (DarkPark) [inline tests/objects/MainPage.js:22]:
```suggestion
        return this.page.getByRole('link', {name: 'GitHub'});
```

File: tests/objects/MainPage.js [hunk 2]
```diff
@@ -123,7 +177,19 @@ export class MainPage {
...
+    async chooseUplinkDirection () {
+        await this.page.getByLabel('uplink').click();
+    }
+
+    async chooseBase64 () {
+        await this.page.getByLabel('base64').click();
+    }
+
+    async getAllTitleTexts () {
+        return await this.page.locator('h5').allInnerTexts();
```
Comment (DarkPark) [inline tests/objects/MainPage.js:189]:
```suggestion
        return this.page.locator('h5').allInnerTexts();
```
and in other places

File: tests/specs/functional.spec.js [hunk 3]
```diff
@@ -1,38 +1,46 @@
...
+        'May be required for parsing and creating a message',
+        '',
+        'Batch processing supported, each dump on a new line',
+        'Commands in the message must be of the same direction'
+    ],
+    analog: {
+        options: commandTypeConfigMap.analog.hardwareTypeList,
+        commands: commandTypeConfigMap.analog.preparedCommandList.map(command => command.value.name.toString())
+    }
+
```
Comment (DeadByElpy) [inline tests/specs/functional.spec.js:19]:
```suggestion
```

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

**Comment by nickkbell** [comment]
closed because not relevant

**Pull request closed without merging**